### ISSUE TITLE
Disable all extension recommendations besides manually defined extensions

### ISSUE THREAD

**Issue by bennettdams** [microsoft/vscode#74137]
### Feature request
It would be nice to have the possiblity to disable all extension recommendations besides the defined extensions from `extensions.json` or workspace settings.

Right now it is only possible to define **specific** unwanted recommendations, which is not sufficient for a multi-root workspace which is used by several developers.

There should be an option for `unwantedRecommendations` like `"all"` as a counterpart to the array.

For example in `extensions.json`:
```
{
  "recommendations": [
      "VisualStudioExptTeam.vscodeintellicode",
    ],
    "unwantedRecommendations": "all"
}
```
The same goes for a multi-root project with a shared `*.code-workspace` , e. g.:

```
"extensions": {
    "recommendations": [
	"VisualStudioExptTeam.vscodeintellicode",
     ],
     "unwantedRecommendations": "all"
}
```

### Background

When using multi-root project which is used by several developers it should be possible to define one workspace-configuration file which contains all parts necessary to setup your development environment. Defining your own extension recommendations and skip all "automatic" recommendations is part of this process, as this ensures that every developer uses the same technology stack.

**Comment by sandy081** [comment]
@bennettdams I think your feature is same as this - https://github.com/microsoft/vscode/issues/57548

**Comment by oliversalzburg** [comment]
Awesome. How nice to return back to this mess after 2 years and seeing it will never be fixed 😂

**Comment by Drkie** [comment]
@sandy081 I'd like to possibly reopen this issue. I don't believe it's a duplicate of https://github.com/microsoft/vscode/issues/57548 and to this day there doesn't seem to be a way to achieve what's described here.

I'll try to explain:
When configuring `extensions.json`, the expectation is that developers in my team will install that exact list of extensions and nothing else - as I want their environment to look as much as mine as possible. VSCode not only recommends all sorts of other extensions, but also mixes them into the "Workspace Recommendations" list, so it becomes impossible to tell what they should install.

Using `extensions.ignoreRecommendations` doesn't work, as this stops **all** recommendations, including the ones I specifically requested to be recommended.
Using `unwantedRecommendations` in `extensions.json` _kind of_ works, however this doesn't stop any future recommendations from taking place - which means I'll always need to keep an eye out for anything out of the ordinary getting recommended so I can add it to this list. Not to mention that for different environments, different things can be recommended, so I won't always be able to tell what will be recommended to someone else.

What's needed in this situation is a setting to stop **every** recommendation, except for the ones listed in `extensions.json` (something like `extensions.ignoreAutomaticRecommendations` - however the `all` entry that OP suggested would work just as well). This means: don't recommend me extensions based on the fact that Docker or WSL is installed, don't recommend me extensions based on files I have opened, and so on. Only recommend me the extensions I listed and nothing else. If possible, I'd also like to remove the "Other recommendations" panel when this setting is enabled.
